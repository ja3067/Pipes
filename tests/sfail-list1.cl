def add(x : list):
	acc = 0
	for i in acc:
		acc = acc + i
	return acc

type(add([1, 2, 3]))
