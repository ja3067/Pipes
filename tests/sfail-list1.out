STypeError: cannot iterate over type int in 'for' loop
